backend/app/api/v1/endpoints/task.py

```diff
@@ -147,9 +148,13 @@ async def search_tasks(
             status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
             detail=f"{SystemMessages.ERROR_FAILED_TO_SEARCH_TASKS} {str(e)}",
         )
-        
-        
-@router.get("/search-delete-requested-tasks/", response_model=TaskList, status_code=status.HTTP_200_OK)
+
+
+@router.get(
+    "/search-delete-requested-tasks/",
+    response_model=TaskList,
+    status_code=status.HTTP_200_OK,
+)
 async def search_delete_requested_tasks(
     query: str,
     skip: int = 0,
```

Comment:
set default value to _limit_ and _skip_ like the following way

```
from fastapi import  Query

limit: int = Query(1, ge=1),
skip: int = Query(0, le=100)
```

Reply:
Done bhaia, 

` skip: int = Query(0, le=5000),`

   ` limit: int = Query(8, le=8),`